src/main/java/org/opentripplanner/transit/model/timetable/OccupancyStatus.java

```diff
@@ -2,34 +2,49 @@
 
 /**
  * OccupancyStatus to be exposed in the API. The values are based on GTFS-RT
- * (transit_realtime.VehiclePosition.OccupancyStatus), but is currently a subset that easily can
- * be mapped to the Nordic SIRI-profile (SIRI 2.1)
- *
- * Descriptions are also based on the SIRI-profile
+ * (transit_realtime.VehiclePosition.OccupancyStatus) that can be easily be mapped to the Nordic
+ * SIRI-profile (SIRI 2.1)
+ * <p>
+ * Descriptions are copied from the GTFS-RT specification.
  */
 public enum OccupancyStatus {
   /**
    * Default. There is no occupancy-data on this departure
    */
-  NO_DATA,
+  NO_DATA_AVAILABLE,
   /**
-   * More than ~50% of seats available
+   * The vehicle is considered empty by most measures, and has few or no passengers onboard, but is
+   * still accepting passengers.
+   */
+  EMPTY,
+  /**
+   * The vehicle or carriage has a large number of seats available. The amount of free seats out of
+   * the total seats available to be considered large enough to fall into this category is
+   * determined at the discretion of the producer.
    */
   MANY_SEATS_AVAILABLE,
   /**
-   * Less than ~50% of seats available
+   * The vehicle or carriage has a small number of seats available. The amount of free seats out of
+   * the total seats available to be considered small enough to fall into this category is
+   * determined at the discretion of the producer.
    */
-  SEATS_AVAILABLE,
+  FEW_SEATS_AVAILABLE,
   /**
-   * Less than ~10% of seats available
+   * The vehicle or carriage can currently accommodate only standing passengers.
    */
   STANDING_ROOM_ONLY,
   /**
-   * Close to or at full capacity
+   * The vehicle or carriage can currently accommodate only standing passengers and has limited
+   * space for them.
+   */
+  CRUSHED_STANDING_ROOM_ONLY,
+  /**
+   * The vehicle is considered full by most measures, but may still be allowing passengers to
+   * board.
    */
   FULL,
   /**
-   * If vehicle/carriage is not in use / unavailable, or passengers are only allowed to alight due to e.g. crowding
+   * The vehicle or carriage is not accepting passengers.
```

Comment:
Please include SIRI documentation as well, . E.g. `are only allowed to alight due to e.g. crowding` is important information. If the GTFS and NeTEx/SIRI descriptions are somewhat different than we should include the differences, not simplify. For the parameters not in Siri, include: `Not supported in SIRI-profile.`

Also, we should include that we encourage merging:
 - EMPTY into MANY_SEATS_AVAILABLE.
 - CRUSHED_STANDING_ROOM_ONLY into FULL.

Reply:
```
/**
   * GTFS: The vehicle or carriage has a large number of seats available. The amount of free seats out
   * of the total seats available to be considered large enough to fall into this category is
   * determined at the discretion of the producer.
   * SIRI: More than ~50% of seats available.
   */
  MANY_SEATS_AVAILABLE,
```

Reply:
Ok that sounds like a good idea, I think previously the comments here were some slightly modified GTFS comments but I wasn't completely sure why.

Reply:
I rechecked the specs. There is SIRI documentation about the occupancy status in two places: https://github.com/SIRI-CEN/SIRI/blob/aca7fa73f8e544f7298b7cb62a823e8dc520ee2a/xsd/siri_model/siri_reference.xsd#L683 and https://enturas.atlassian.net/wiki/spaces/PUBLIC/pages/637370392/SIRI-ET#SIRI-ET-VehicleOccupancyStructure . Which one should we use here @t2gran?

Reply:
When you say that we encourage the merging, is that meant for data data producers or do you want to merge these values in OTP?

I'm in favour of reducing the number of enum values in the data model and in the APIs, since I'm off the opinion that adding enum values later is not a backwards-incompatible change.

Reply:
I think we want to encourage the client software to merge these. The merging code is simply to implement anywhere anyone wants and I think it's a bit cleaner. For example, @t2gran interpretation of what should be merged together is slightly different from what the spec was for us in our project:
    TransitData Empty = DigiTransit level 1
    TD Many_seats_available = DT level 1
    TD Few_seats_available = DT level 2
    TD Standing_room_only = DT level 3
    TD Crushed_standing_room_only = DT level  4
    TD Full = DT level 5
    TD Not_accepting_passengers = DT level 5